src/remoteSync/s3/validator.js

```diff
@@ -0,0 +1,27 @@
+const files = require('./../../utils/files');
+
+const s3ConfigKeys = {
+    s3AccKeyLoc: 's3sdkPath',
```

Comment:
Change to user understandable name. ("s3sdkPath" to "s3CredentialsFilePath" maybe).